### PR TITLE
vita3k: Fix settings grammar

### PR DESCRIPTION
**Pull request by ArbestRi**
I found some problems in here with the grammar in the configuration menu and it might need to be fixed

### CODE REVIEW

**Review by EXtremeExploit — DISMISSED** [review]

File: vita3k/gui/src/settings_dialog.cpp [hunk 1]
```diff
@@ -455,7 +455,7 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Spacing();
 #endif
         ImGui::SetCursorPosX((ImGui::GetWindowWidth() / 2.f) - (ImGui::CalcTextSize("Resolution Upscaling").x / 2.f));
-        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Resolution Upscaling");
+        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Internal Resolution Upscaling");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:458]: wrong nee dchange for both, look upp line

File: vita3k/gui/src/settings_dialog.cpp [hunk 2]
```diff
@@ -407,7 +407,7 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
             }
             ImGui::SameLine();
         } else {
-            ImGui::TextColored(GUI_COLOR_TEXT, "No modules present.\nPlease download and install the last firmware.");
+            ImGui::TextColored(GUI_COLOR_TEXT, "No modules present.\nPlease download and install the firmware.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:410]: wrong, is last firmwre

File: vita3k/gui/src/settings_dialog.cpp [hunk 3]
```diff
@@ -493,14 +493,14 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Spacing();
         ImGui::Checkbox("Use shader cache", &host.cfg.shader_cache);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Enable shader cache to pre-compile it at boot up\nUncheck the box to disable this feature.");
+            ImGui::SetTooltip("Enables shader cache to pre-compile it at boot up\nUncheck the box to disable this feature.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:496]: why s ?

File: vita3k/gui/src/settings_dialog.cpp (hunk 1 above)
Comment (EXtremeExploit) [reply vita3k/gui/src/settings_dialog.cpp:458]: The upscaling is indeed by altering the internal resolution, change is correct

File: vita3k/gui/src/settings_dialog.cpp [hunk 4]
```diff
@@ -623,7 +623,7 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::SameLine();
         ImGui::Checkbox("Info Bar Visible", &host.cfg.show_info_bar);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Check the box to show info bar inside app selector.");
+            ImGui::SetTooltip("Check the box to show a info bar inside app selector.");
```
Comment (EXtremeExploit) [inline vita3k/gui/src/settings_dialog.cpp:626]: an

File: vita3k/gui/src/settings_dialog.cpp [hunk 5]
```diff
@@ -528,7 +528,7 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Spacing();
         ImGui::Checkbox("PS TV Mode", &config.pstv_mode);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Check the box to enable PS TV Emulated mode.");
+            ImGui::SetTooltip("Check this box to enable PS TV mode.");
```
Comment (EXtremeExploit) [inline vita3k/gui/src/settings_dialog.cpp:531]: the this isn't really needed, you can say "Check to enable PS TV mode"

File: vita3k/gui/src/settings_dialog.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp:458]: i no speek this, i speek need change two line..., if you change string, need change also calctextsize line

File: vita3k/gui/src/settings_dialog.cpp (hunk 2 above)
Comment (EXtremeExploit) [reply vita3k/gui/src/settings_dialog.cpp:410]: ##### could have various correct sentences like
* install the latest firmware
* install the PSVita firmware
* install firmware(?
##### im not sure about "install the last firmware" as it doesn't sound right

File: vita3k/gui/src/settings_dialog.cpp [hunk 6]
```diff
@@ -455,7 +455,7 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Spacing();
 #endif
         ImGui::SetCursorPosX((ImGui::GetWindowWidth() / 2.f) - (ImGui::CalcTextSize("Resolution Upscaling").x / 2.f));
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:457]: you need change here too i said

File: vita3k/gui/src/settings_dialog.cpp [hunk 7]
```diff
@@ -520,15 +520,15 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
     if (ImGui::BeginTabItem("System")) {
         ImGui::PopStyleColor();
         ImGui::Spacing();
-        ImGui::TextColored(GUI_COLOR_TEXT, "Enter Button Assignment \nSelect your 'Enter' Button.");
+        ImGui::TextColored(GUI_COLOR_TEXT, "Enter button assingment \nSelect your 'Enter' button.");
```
Comment (Macdu) [inline vita3k/gui/src/settings_dialog.cpp:523]: assignment

File: vita3k/gui/src/settings_dialog.cpp (hunk 6 above)
Comment (ArbestRi) [reply vita3k/gui/src/settings_dialog.cpp:457]: Might look something vital

File: vita3k/gui/src/settings_dialog.cpp [hunk 8]
```diff
@@ -750,13 +750,13 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Save color surfaces to files.");
         ImGui::Spacing();
-        ImGui::Checkbox("Dump textures", &host.cfg.dump_textures);
+        ImGui::Checkbox("Texture dumping", &host.cfg.dump_textures);
```
Comment (EXtremeExploit) [inline vita3k/gui/src/settings_dialog.cpp:753]: im not a very big fan of this change, if you stand by this change then the other elements like "Log exports" should turn into "Exports logging". Also if you have time you can correct the capitalization on that menu too (the debug one, the one that has Log Exports)

File: vita3k/gui/src/settings_dialog.cpp [hunk 9]
```diff
@@ -750,13 +750,13 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
...
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Dump textures to files");
+            ImGui::SetTooltip("Dumps textures to files");
         ImGui::SameLine();
-        ImGui::Checkbox("Dump elfs", &host.kernel.debugger.dump_elfs);
+        ImGui::Checkbox("ELF dumping", &host.kernel.debugger.dump_elfs);
```
Comment (EXtremeExploit) [inline vita3k/gui/src/settings_dialog.cpp:757]: Same here as above, either "Dump ELFs" or change the rest of the debug tab to use that kind of sentece structure

**Review by EXtremeExploit — CHANGES_REQUESTED** [review]

File: vita3k/gui/src/settings_dialog.cpp (hunk 8 above)
Comment (ArbestRi) [reply vita3k/gui/src/settings_dialog.cpp:753]: It looks like I fixed all the capitalization on the whole Debug menu while not touching anything that looks vital to the menu.

File: vita3k/gui/src/settings_dialog.cpp (hunk 9 above)
Comment (EXtremeExploit) [reply vita3k/gui/src/settings_dialog.cpp]: change to "ELF Dumping" to use same sentence structure as the rest checkboxes and should be good to merge

Comment (ArbestRi) [reply vita3k/gui/src/settings_dialog.cpp]: I did this

**Review by EXtremeExploit — APPROVED** [review]
LGTM :+1:

File: vita3k/gui/src/settings_dialog.cpp [hunk 10]
```diff
@@ -359,9 +359,9 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::PopStyleColor();
         ImGui::Spacing();
         if (!gui.modules.empty()) {
-            ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Modules Mode");
+            ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "Modules mode");
             ImGui::Spacing();
-            ImGui::Checkbox("Experimental: LLE DriverUser", &config.lle_driver_user);
+            ImGui::Checkbox("LLE DriverUser", &config.lle_driver_user);
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:364]: fail, delete this change

File: vita3k/gui/src/settings_dialog.cpp [hunk 11]
```diff
@@ -407,7 +407,7 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
             }
             ImGui::SameLine();
         } else {
-            ImGui::TextColored(GUI_COLOR_TEXT, "No modules present.\nPlease download and install the last firmware.");
+            ImGui::TextColored(GUI_COLOR_TEXT, "No modules present.\nPlease download and install the PS Vita firmware.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:410]: i said here, keep LAST firmware

File: vita3k/gui/src/settings_dialog.cpp [hunk 12]
```diff
@@ -424,16 +424,16 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
...
             config.cpu_backend = LIST_CPU_BACKEND[int(config_cpu_backend)];
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Select your preferred cpu backend.");
+            ImGui::SetTooltip("Select your preferred CPU backend.");
         if (config_cpu_backend == CPUBackend::Dynarmic) {
             ImGui::Spacing();
             ImGui::Checkbox("Enable optimizations", &config.cpu_opt);
             if (ImGui::IsItemHovered())
-                ImGui::SetTooltip("Enable additional CPU JIT optimizations.");
+                ImGui::SetTooltip("Enables additional CPU JIT optimizations.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:436]: Why S ?

File: vita3k/gui/src/settings_dialog.cpp [hunk 13]
```diff
@@ -480,11 +480,11 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
             ImGui::EndDisabled();
         ImGui::Checkbox("Disable surface sync", &config.disable_surface_sync);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Speed hack, disable surface syncing between cpu and gpu.\nSurface syncing is needed by a few games.\nGive a big performance boost if disabled (in particular when upscaling is on).");
+            ImGui::SetTooltip("Speed hack, disables surface syncing between CPU and GPU.\nSurface syncing is needed by a few games.\nGives a big performance boost if disabled (in particular when upscaling is on).");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:483]: why you add s every where ?

File: vita3k/gui/src/settings_dialog.cpp [hunk 14]
```diff
@@ -480,11 +480,11 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
...
         ImGui::Spacing();
         ImGui::Checkbox("Enable anti-aliasing (FXAA)", &config.enable_fxaa);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Anti-aliasing is a technique for smoothing out jagged edges.\n FXAA comes at almost no performance cost but makes the game look slightly blurry.");
+            ImGui::SetTooltip("Anti-aliasing is a technique for smoothing out jagged edges.\n FXAA comes at almost no performance cost but makes games look slightly blurry.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:487]: revert is fine like that

File: vita3k/gui/src/settings_dialog.cpp [hunk 15]
```diff
@@ -493,14 +493,14 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Spacing();
         ImGui::Checkbox("Use shader cache", &host.cfg.shader_cache);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Enable shader cache to pre-compile it at boot up\nUncheck the box to disable this feature.");
+            ImGui::SetTooltip("Enables shader cache to pre-compile it at game startup\nUncheck to disable this feature.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:496]: no S, keep box word

File: vita3k/gui/src/settings_dialog.cpp [hunk 16]
```diff
@@ -520,15 +520,15 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
...
+        ImGui::TextColored(GUI_COLOR_TEXT, "Enter button assignment \nSelect your 'Enter' button.");
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("This is the button that is used as 'Confirm' in applications dialogs. \nSome applications don't use this and get default confirmation button.");
         ImGui::RadioButton("Circle", &host.cfg.sys_button, 0);
         ImGui::RadioButton("Cross", &host.cfg.sys_button, 1);
         ImGui::Spacing();
         ImGui::Checkbox("PS TV Mode", &config.pstv_mode);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Check the box to enable PS TV Emulated mode.");
+            ImGui::SetTooltip("Check to enable PS TV mode.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:531]: check what ? no remove box, revert full line, is emulated mode

File: vita3k/gui/src/settings_dialog.cpp [hunk 17]
```diff
@@ -551,20 +551,20 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Spacing();
         ImGui::Checkbox("Archive Log", &host.cfg.archive_log);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Check the box to enable Archiving Log.");
+            ImGui::SetTooltip("Enables log archiving.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:554]: revert

File: vita3k/gui/src/settings_dialog.cpp [hunk 18]
```diff
@@ -581,22 +581,22 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
...
         ImGui::PushItemWidth(320);
         ImGui::TextColored(GUI_COLOR_TEXT, "Current emulator folder: %s", host.cfg.pref_path.c_str());
         ImGui::PopItemWidth();
         ImGui::Spacing();
-        if (ImGui::Button("Change Emulator Path"))
+        if (ImGui::Button("Change emulator path"))
             change_emulator_path(gui, host);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Change Vita3K emulator folder path.\nYou will need to move your old folder to the new location manually.");
+            ImGui::SetTooltip("Changes Vita3K emulator folder path.\nYou will need to move your old folder to the new location manually.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:596]: why S ?

File: vita3k/gui/src/settings_dialog.cpp [hunk 19]
```diff
@@ -606,7 +606,7 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
                 }
             }
             if (ImGui::IsItemHovered())
-                ImGui::SetTooltip("Reset Vita3K emulator path to default.\nYou will need to move your old folder to the default location manually.");
+                ImGui::SetTooltip("Resets Vita3K emulator path to default.\nYou will need to move your old folder to the default location manually.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:609]: is one reset, no S

File: vita3k/gui/src/settings_dialog.cpp [hunk 20]
```diff
@@ -641,20 +641,20 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
...
+        const auto font_size = ImGui::CalcTextSize("Font support").x;
         ImGui::SetCursorPosX((ImGui::GetWindowWidth() / 2.f) - (font_size / 2.f));
-        ImGui::TextColored(GUI_COLOR_TEXT_MENUBAR, "Font Support");
+        ImGui::TextColored(GUI_COLOR_TEXT_MENUBAR, "Font support");
         ImGui::Spacing();
         if (gui.fw_font) {
             ImGui::Checkbox("Asia Region", &host.cfg.asia_font_support);
             if (ImGui::IsItemHovered())
-                ImGui::SetTooltip("Check this box to enable font support for Korean and Traditional Chinese.\nEnabling this will use more memory and will require you to restart the emulator.");
+                ImGui::SetTooltip("Enables font support for Korean and Traditional Chinese.\nEnabling this will use more memory and will require you to restart the emulator.");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:651]: keep check the box

File: vita3k/gui/src/settings_dialog.cpp [hunk 21]
```diff
@@ -730,33 +730,33 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
     if (ImGui::BeginTabItem("Debug")) {
         ImGui::PopStyleColor();
         ImGui::Spacing();
-        ImGui::Checkbox("Log Imports", &host.kernel.debugger.log_imports);
+        ImGui::Checkbox("Import logging", &host.kernel.debugger.log_imports);
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:733]: donj't touch on debug option revert all

File: vita3k/gui/src/settings_dialog.cpp [hunk 22]
```diff
@@ -730,33 +730,33 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
...
         ImGui::SameLine();
         ImGui::Checkbox("Save color surfaces", &host.cfg.color_surface_debug);
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Save color surfaces to files.");
         ImGui::Spacing();
-        ImGui::Checkbox("Dump textures", &host.cfg.dump_textures);
+        ImGui::Checkbox("Texture dumping", &host.cfg.dump_textures);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Dump textures to files");
+            ImGui::SetTooltip("Dumps textures to files");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:755]: dump no have s

**Review by Zangetsu38 — CHANGES_REQUESTED** [review]

File: vita3k/gui/src/settings_dialog.cpp (hunk 10 above)
Comment (EXtremeExploit) [reply vita3k/gui/src/settings_dialog.cpp:364]: Not necessary to delete this change, its saying its experimental in the tooltip, no need to clutter the screen

File: vita3k/gui/src/settings_dialog.cpp (hunk 11 above)
Comment (EXtremeExploit) [reply vita3k/gui/src/settings_dialog.cpp:410]: Any firmware is usable in the emulator, also how can someone not install the latest when the button only allows you to download the latest one

File: vita3k/gui/src/settings_dialog.cpp (hunk 10 above)
Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp:364]: is needed, some user aleady enable it and after complain :/

File: vita3k/gui/src/settings_dialog.cpp (hunk 13 above)
Comment (EXtremeExploit) [reply vita3k/gui/src/settings_dialog.cpp:483]: thats how english works

Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp:483]: ? english no have s for enable and disable

File: vita3k/gui/src/settings_dialog.cpp (hunk 14 above)
Comment (EXtremeExploit) [reply vita3k/gui/src/settings_dialog.cpp:487]: The setting is applied to every game, so its better to say "games" as saying "the game" implies that only applies to a single game

Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp:487]: yes, macdu have writed this for said current game you run or want run

Comment (EXtremeExploit) [reply vita3k/gui/src/settings_dialog.cpp:487]: minor mistake, can happen to anyone

Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp:487]: ok

File: vita3k/gui/src/settings_dialog.cpp (hunk 12 above)
Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp:436]: this sentence literally doesn't mean the same thing anymore
when you adding S it no said "activate additonal" but allow additionnal, is totaly wrong

**Pull request merged**